es6.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  parserOptions: {
-    ecmaVersion: 6,
+  env: {
+    es6: true,
```

Comment:
I assume this doesn't require any code changes on our stuff that uses the es6 config?